diff --git a/backend/config/settings.py b/backend/config/settings.py
index 66e6f863..0caeb497 100644
--- a/backend/config/settings.py
+++ b/backend/config/settings.py
@@ -80,7 +80,7 @@ DATABASES = {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'bid_db',
         'USER': 'root',
-        'PASSWORD': 'Sunil@123',
+        'PASSWORD': 'varsha@11',
         'HOST': 'localhost',
         'PORT': '3306',
     }
